Google.PowerShell.IntegrationTests/CloudResourceManager/CloudResourceManager.IamPolicy.Tests.ps1

```diff
@@ -0,0 +1,20 @@
+﻿. $PSScriptRoot\..\GcloudCmdlets.ps1
+Install-GcloudCmdlets
+
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-GceInstance" {
+    It "should work" {
+        $policies = Get-GcIamPolicy
+        $policies.Count -gt 0 | Should Be $true
+
+        $editors = $policies | Where-Object {$_.Role -eq "roles/editor"}
+        $editors.Members.Count -gt 0 | Should Be $true
+        $editors.Members -contains "serviceAccount:appveyorci-testing@gcloud-powershell-testing.iam.gserviceaccount.com" |
```

Comment:
What is this? appveyorci-testing@gcloud-powershell-testing.iam.gserviceaccount.com

Can we add this as a const and add a comment?